test/functional/rpc_blockchain.py

```diff
@@ -388,6 +390,33 @@ def _test_gettxoutsetinfo(self):
         # Unknown hash_type raises an error
         assert_raises_rpc_error(-8, "'foo hash' is not a valid hash_type", node.gettxoutsetinfo, "foo hash")
 
+    def _test_gettxout(self):
+        self.log.info("Test gettxout")
+        node = self.nodes[0]
+
+        # Get the best block hash and the block, which
+        # should only include the coinbase transaction.
+        best_block_hash = node.getbestblockhash()
+        block = node.getblock(best_block_hash)
+        assert_equal(block['nTx'], 1)
+
+        # Get the transaction ID of the coinbase tx and
+        # the transaction output.
+        txid = block['tx'][0]
+        txout = node.gettxout(txid, 0)
+
+        # Validate the gettxout response
+        assert_equal(txout['bestblock'], best_block_hash)
+        assert_equal(txout['confirmations'], 1)
+        assert_equal(txout['value'], create_coinbase(HEIGHT).vout[0].nValue / 100000000)
```

Comment:
Should we use `COIN` here instead of `100000000`

found in `bitcoin/test/functional/test_framework/messages.py`

Reply:
Done. Thanks